### PR TITLE
created tiddlywiki template

### PR DESCRIPTION
**Pull request by bravohotel91**
Created tiddlywiki template in response to Issue #356

### CODE REVIEW

**Comment by nwithan8** [comment]
Thanks for tackling a bunch of these open issues @bravohotel91 !

I'll add this to the PR template, but moving forward, before taking up a task, can you check if a version of the app already exists in Community Apps? For this one: https://unraid.net/community/apps/c/mediaserver?q=Comixed&r=0#r

While we'd love to be the go-to location for all the templates for these apps, we also don't want to flood Community Apps with duplicates if someone else is already maintaining a template for a particular app.

Comixed has a template already maintained by another user: https://github.com/C3004/Unraid-Templates-C3004/blob/main/Comixed-Docker.xml

That said, this template hasn't been updated in over 2 years. Meanwhile, a new version of Comixed was released just 4 days ago. I don't know if the existing template is out of date for Comixed's current application.

If you can compare your template against this one and let us know if there are significant-enough changes that warrant us adopting a Comixed template.

**Comment by bravohotel91** [comment]
@nwithan8 - Sorry i must've missed that. That template is still valid and using the latest docker image. We can cancel the purge request

**Comment by nwithan8** [comment]
> @nwithan8 - Sorry i must've missed that one but this one is for the tiddlywiki template. The one that is currently in the CA is not the official one. My template uses the official docker app for tiddlywiki per the project's documentation.

Oops, my bad. That whole reply, and I post it on the wrong thread 🙃

File: templates/tiddlywiki.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  <Privileged>false</Privileged>
+  <Support>https://hub.docker.com/r/elasticdog/tiddlywiki/</Support>
+  <Project>https://github.com/Jermolene/TiddlyWiki5</Project>
+  <Overview>TiddlyWiki is a self-contained JavaScript wiki, configured here to run as a Node.js application.&#xD;
+&#xD;
+Converted By Community Applications   Always verify this template (and values)  against the support page for the container&#xD;
+&#xD;
+</Overview>
+<WebUI>http://[IP]:[PORT:8080]</WebUI>
+  <Config Name="AppData Location" Target="/tiddlywiki" Default="/mnt/user/appdata/tiddlywiki" Mode="rw" Description="" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/appdata/tiddlywiki</Config>
```
Comment (nwithan8) [inline templates/tiddlywiki.xml:17]:
```suggestion
  <Config Name="AppData Location" Target="/tiddlywiki" Default="/mnt/user/appdata/tiddlywiki" Mode="rw" Description="" Type="Path" Display="always" Required="true" Mask="false">/mnt/user/appdata/tiddlywiki</Config>
```

Let's avoid in-container-only data, could inadvertently lead to users bloating their `docker.img`

File: templates/tiddlywiki.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  <Config Name="WebUI" Target="8080" Default="8080" Mode="tcp" Description="" Type="Port" Display="always" Required="false" Mask="false">8080</Config>
```
Comment (nwithan8) [inline templates/tiddlywiki.xml:18]:
```suggestion
  <Config Name="WebUI" Target="8080" Default="8080" Mode="tcp" Description="" Type="Port" Display="always" Required="true" Mask="false">8080</Config>
```

**Review by nwithan8 — APPROVED** [review]

**Review by nwithan8 — CHANGES_REQUESTED** [review]
See comments.

**Comment by bravohotel91** [comment]
Thanks for catching those.

**Review by nwithan8 — APPROVED** [review]

**Pull request merged**